setup.cfg

```diff
@@ -83,18 +83,17 @@ testpaths = "sunpy" "docs"
 norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" "sunpy[/\]_dev" ".jupyter" ".history" "tools" "sunpy[\/]extern"
 doctest_plus = enabled
 doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
-addopts = --doctest-rst -m "not figure" --doctest-ignore-import-errors
+addopts = --doctest-rst -m "not mpl_image_compare" --doctest-ignore-import-errors
```

Comment:
I don't think we need this. pytest-mpl skips if the `--mpl` flag isn't specified. As long as we can assume pytest-mpl is installed we don't need this. If we don't want to assume that then it's probably better to keep it.

Reply:
Fixed.